File: WordApp/ClientApp/src/app/users/user.model.ts
export class UserModel{
  name: string;
  password: string;
  userType: string;
  id: string;
  rowVersion: string;

  constructor(name: string, password: string, userType: string, id: string, rowVersion: string) {
    this.name = name;
    this.password = password;
    this.userType = userType;
    this.id = id;
    this.rowVersion = rowVersion;
  }
}

File: WordApp/Models/User/UserModel.cs
using Entities.Enums;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using WordApp.Models.Base;

namespace WordApp.Models.User
{
    public class UserModel: BaseModel
    {
        public string Name { get; set; }
        public string Password { get; set; }

        [JsonConverter(typeof(StringEnumConverter))]
        public UserType UserType { get; set; }
        public UserTokenModel Token { get; set; }
    }
}
